article/performance-comparisons.tex

```diff
@@ -112,3 +112,39 @@ \subsubsection{Hyperparameter experiment}
     \caption{Recall$@$100k on NGCF PAS with different input values}
     \label{fig:recall-ngcfpas-weights}
 \end{figure}
+
+\subsubsection{One convolution experiment}
+An experiment was conducted to see the influence of only using one convolution in the different methods.
```

Comment:
```suggestion
An experiment was conducted to see what influence it would have on the performance of the methods to change the number of convolutions to one.
```